apps/nextjs-website/__tests__/helpers/productHeader.helpers.test.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable functional/no-expression-statements */
+import { productToMenuItems } from '@/helpers/productHeader.helper';
+import { Product } from '@/lib/types/product';
+
+const product: Product = {
+  name: 'test',
+  path: '/path',
+  subpaths: {
+    overview: {
+      name: 'overview',
+      path: '/path/overview_path',
+    },
+    tutorial: {
+      name: 'tutorial',
+      path: '/path/tutorial_path',
+    },
+  },
+};
+
+it('should convert product to menu items', () => {
+  const themeLight = 'light';
+  const menuItems = productToMenuItems(
+    product,
+    '/path/overview_path',
+    themeLight
+  );
+  expect(menuItems.length).toEqual(2);
+  const overviewItem = menuItems.find(
+    ({ href }) => href === product.subpaths.overview.path
+  );
+  expect(!!overviewItem).toBeTruthy;
+  expect(overviewItem?.label).toBe(product.subpaths.overview.name);
+  expect(overviewItem?.href).toBe(product.subpaths.overview.path);
+  expect(overviewItem && overviewItem?.active).toBeTruthy;
+  expect(overviewItem?.theme).toBe(themeLight);
```

Comment:
```suggestion
  expect(overviewItem).not.toBeUndefined();
  expect(overviewItem?.label).toBe(product.subpaths.overview.name);
  expect(overviewItem?.href).toBe(product.subpaths.overview.path);
  expect(overviewItem?.active).toBeTruthy();
  expect(overviewItem?.theme).toBe(themeLight);
```

main issue here was the `toBeTruthy` called without the `()`: if you replace the `toBeTruthy` with `toBeFalsy` tests pass.

Reply:
thanks i missed it